CHANGELOG.md

```diff
@@ -10,7 +10,10 @@
 
 * [#8810](https://github.com/rubocop-hq/rubocop/pull/8810): Fix multiple offense detection for `Style/RaiseArgs`. ([@pbernays][])
 * [#8809](https://github.com/rubocop-hq/rubocop/pull/8809): Fix multiple offense detection for `Style/For`. ([@pbernays][])
-* [#8801](https://github.com/rubocop-hq/rubocop/pull/8801): Fix `Layout/SpaceAroundEqualsInParameterDefault` only registered once in a line. ([@rdunlop][])
+* [#8801](https://github.com/rubocop-hq/rubocop/issues/8801): Fix `Layout/SpaceAroundEqualsInParameterDefault` only registered once in a line. ([@rdunlop][])
+* [#8810](https://github.com/rubocop-hq/rubocop/pull/8810): Fix multiple offense detection for Style/RaiseArgs. ([@pbernays][])
+* [#8809](https://github.com/rubocop-hq/rubocop/pull/8809): Fix multiple offense detection for Style/For. ([@pbernays][])
+* [#8799](https://github.com/rubocop-hq/rubocop/issues/8799): Fix `Layouts/SpaceInsideBlockBraces` regression. ([@rdunlop][])
```

Comment:
It seems that unexpected changelog entries has been updated because it has been away from the master branch changing. Can you rebase with the latest master branch again? I'm sorry for taking up your time 💦

Reply:
oops, sorry, I must have messed. that up.

Happy to iterate.